packages/js/package.json

```diff
@@ -111,6 +111,7 @@
     "socket.io-client": "4.7.2",
     "solid-floating-ui": "^0.3.1",
     "solid-js": "^1.8.11",
+    "solid-motionone": "^1.0.1",
```

Comment:
How much overhead does it add to the bundle size?

Reply:
@SokratisVidros 
https://bundlephobia.com/package/@motionone/solid@10.16.4
16.3kB minified, 6.5kB minified + gzipped